docs/user-guide/connectors/custom-connectors.rst

```diff
@@ -31,6 +31,28 @@ To send a message, you would run a command like:
 
 where ``myio`` is the name of your component.
 
```

Comment:
Maybe we should even set up section headers in this doc? e.g.

Custom Connectors
- Channel setup
- Sending messages

and reorganize based on that -- thoughts @wochinge ?

Reply:
ok, separate issue then and merge this @erohmensing ?

Reply:
+ removing the example for `To send a message with metadata` since it's more confusing than helpful.

Reply:
sure thing, will create an issue